o3po/includes/class-o3po-settings.php

```diff
@@ -1647,6 +1667,7 @@ public function get_all_settings_fields_map() {
                 'journal_subtitle' => 'trim_settings_field',
                 'journal_description' => 'trim_settings_field',
                 'journal_level_doi_suffix' => 'validate_doi_suffix',
+                'doi_suffix_template' => 'trim_settings_field',
```

Comment:
More careful validation is necessary here. Rember that this is not just used as part of the DOI but also in the slugs of individual particle pages. The result of the substitution must be a valid slug.

Reply:
Also this has been taken care of as part of the refactoring